main/src/ca/uwaterloo/flix/language/phase/Instances.scala

```diff
@@ -248,13 +248,6 @@ object Instances {
     * Reassembles an instance
     */
   private def checkInstance(inst: TypedAst.Instance, root: TypedAst.Root, changeSet: ChangeSet)(implicit flix: Flix): List[InstanceError] = {
-    val isTraitStable = inst.trt.loc.source.input.isStable
```

Comment:
I went with the safe option here: This just means we end up re-checking everything.